chat-plugins/roomsettings.js

```diff
@@ -122,6 +122,14 @@ class RoomSettings {
 			return this.button('%', null, 'tournament enable %') + this.button('@', null, 'tournament enable @') + this.button('#', true);
 		}
 	}
+	r9k() {
+		if (!this.user.can('editroom', null, this.room)) return this.button(this.room.r9k ? 'on' : 'off', true);
+		if (this.room.r9k) {
+			return this.button('off', null, 'r9k off') + this.button('on', true);
```

Comment:
I would probably call the states `Robot9000 filtering` and `off`. I think that matches the others the most?